examples/lighting-app/infineon/cyw30739/args.gni

```diff
@@ -20,7 +20,7 @@ cyw30739_sdk_target = get_label_info(":sdk", "label_no_toolchain")
 
 chip_openthread_ftd = true
 
-chip_enable_ota_requestor = true
+chip_enable_ota_requestor = false
```

Comment:
There are only a few hundred bytes left in CYW30739 flash.
I need to do something before the CYW30739 building breaks CI.
I just enabled OTA and disabled FTD, this may be a more proper configuration.